corehq/apps/data_analytics/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+
+class MALTRow(models.Model):
```

Comment:
I just realized this but you should also have a combined unique constraint for `month`, `user_id`, and `domain_name`